New post: Server softwares (#43)
FalixNodes-Software/help-center: _posts/minecraft/general/2021-08-22-server-softwares.md

---
layout: post
title:  "Server Softwares"
categories: Minecraft
tag: mcgeneral
permalink: /minecraft/general/server-softwares/
---

# Server Softwares
Choosing your server's software is a very important task. This article will help you choose the software that fits your server's needs.

## PaperMC

> [PaperMC](https://papermc.io) is a fork of Spigot that is optimized and allows you to use Spigot and Bukkit plugins. It also fixes some Vanilla bugs and exploits!

Reviewer: No need to mention what plugins you can run imo.
If you already mention Paper is a fork of Spigot, then it makes sense you can run Spigot/Bukkit plugins.

```suggestion
[PaperMC](https://papermc.io) is an optimized fork of Spigot that fixes some Vanilla bugs and exploits!
```

## Purpur

> [Purpur](https://purpur.pl3x.net/) is a Fork of Tuinity that is very optimized. It contains extra features and supports Spigot and Bukkit plugins. We recommend using this server software.

Reviewer: ```suggestion
[Purpur](https://purpur.pl3x.net/) is an optimized Fork of Tuinity. It contains extra features and supports Spigot and Bukkit plugins. We recommend using this.
```

> ## Fabric
> [Fabric](https://fabricmc.net/) is a lightweight Minecraft software used for Fabric mods.

Reviewer: ```suggestion
## FabricMC
[FabricMC](https://fabricmc.net) is a lightweight Minecraft software used for Fabric mods.
```
